oshmem/mca/sshmem/verbs/configure.m4

```diff
@@ -37,6 +37,49 @@ AC_DEFUN([MCA_oshmem_sshmem_verbs_CONFIG],[
     AC_SUBST([oshmem_verbs_LDFLAGS])
     AC_SUBST([oshmem_verbs_LIBS])
 
+    oshmem_have_exp_reg_shared_mr=0
```

Comment:
looking at the m4 file, seems like this check(AC_COMPILE_IFELSE) should replace these lines:
```
            AC_CHECK_DECLS([IBV_EXP_ACCESS_ALLOCATE_MR,IBV_EXP_ACCESS_SHARED_MR_USER_READ],
                   [oshmem_have_mpage=3], [],
                   [#include <infiniband/verbs.h>])
```

because it's also possible that the "old" API of shared_mr is supported (oshmem_have_mpage=2) and then no need to disable the verbs component